internal/k8s/exploding_client.go

```diff
@@ -102,3 +104,11 @@ func (ec *explodingClient) OwnerFetcher() OwnerFetcher {
 func (ec *explodingClient) ClusterHealth(_ context.Context, _ bool) (ClusterHealth, error) {
 	return ClusterHealth{}, errors.Wrap(ec.err, "could not set up kubernetes client")
 }
+
+func (ec *explodingClient) ConnectionStatus() *v1alpha1.KubernetesClusterConnectionStatus {
+	return &v1alpha1.KubernetesClusterConnectionStatus{
+		Context:   "default",
+		Namespace: "default",
+		Product:   string(clusterid.ProductUnknown),
+	}
```

Comment:
I wonder if this should be `nil` or empty object?

Reply:
i made context and namespace empty